internal/config/config_test.go

```diff
@@ -41,10 +41,10 @@ func (s *ConfigTestSuite) AddRequiredFlags() {
 func (s *ConfigTestSuite) TestExtraArguments() {
 	s.AddRequiredFlags()
 	s.NoError(s.cmd.Flags().Set("version-analyzer-name", "helm"))
-	s.NoError(s.cmd.Flags().Set("helm-"+helm.ArgumentKeyRootDir, "test-value"))
+	s.NoError(s.cmd.Flags().Set("helm-"+versionanalyzers.ArgumentKeyRootDir, "test-value"))
```

Comment:
That shows it should be renamed to s.th. like:
```suggestion
	s.NoError(s.cmd.Flags().Set("helm-"+versionanalyzers.HelmArgumentKeyRootDir, "test-value"))
```
To be distinguishable from other analyzers...